### DISCUSSION TITLE
Add PiP in Tab Sidebar

### DISCUSSION THREAD

**Discussion by jackgraddon** [zen-browser/desktop#4064] [Ideas]
Incorporating a Now Playing bar with PiP support into the tab sidebar of Zen would be a cool addition, and would help add feature parity to Arc Browser, where Zen gets many ideas from. Not only would it mirror Arc's [Mini Player](https://resources.arc.net/hc/en-us/articles/19234766331799-Mini-Player-Watch-or-Listen-as-you-Browse), I believe it could extend its usefulness by adding the video or album cover to the sidebar along with the playback controls.

Here is an example made with PiP:
<img width="479" alt="SCR-20241229-kghc" src="https://github.com/user-attachments/assets/f42cf001-c95f-4b2c-9b5b-64ff55ecfcac" />

**Comment by udaygirhepunje** [comment]
Super cool Idea
